app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (NotWearingMaskError,
+                        OutdatedVaccineError,
+                        NotVaccinatedError)

```

Comment:
The `NotWearingMaskError` class should inherit from `VaccineError` instead of `Exception` to maintain consistency with the other error classes and ensure proper exception handling. Please update the inheritance in the `errors.py` file accordingly.